covalent_dispatcher/_core/execution.py

```diff
@@ -611,6 +639,85 @@ async def _postprocess_workflow(result_object: Result) -> Result:
     return result_object
 
 
+async def _reconstruct_sublattice_result(result_object: Result) -> Result:
+    """
+    Reconstruct a sublattice workflow result by making electron output substitutions.
+
+    Args:
+        result_object: Result object being used for current dispatch
+
+    Returns:
+        The result object with a restructured result value
+    """
+    rc_executor = result_object.lattice.get_metadata("workflow_executor")
+    rc_executor_data = result_object.lattice.get_metadata("workflow_executor_data")
+    selected_executor = [rc_executor, rc_executor_data]
+
+    result_object._status = Result.PENDING_RECONSTRUCTION
+    upsert._lattice_data(result_object)
+
+    app_log.debug(f"Preparing to reconstruct sublattice result {result_object.dispatch_id}")
+
+    reconstruct_inputs = {
+        "args": [TransportableObject.make_transportable(result_object._lattice)],
+        "kwargs": {},
+    }
+
+    try:
+        future = asyncio.create_task(
+            _run_task(
+                result_object=result_object,
+                node_id=-2,
+                serialized_callable=TransportableObject(_reconstruct),
+                selected_executor=selected_executor,
+                node_name="reconstruct",
+                call_before=[],
+                call_after=[],
+                inputs=reconstruct_inputs,
+                workflow_executor=selected_executor,
+            )
+        )
+        rc_start_time = datetime.now(timezone.utc)
+        app_log.debug(
+            "Submitting reconstruction job to executor " f"{selected_executor} at {rc_start_time}"
+        )
+
+        rc_result = await future
+    except Exception as ex:
+        # Error during dispatch, no useful result
+        app_log.debug(f"Exception during reconstruction: {ex}")
+        result_object._result = TransportableObject(None)
+        result_object._status = Result.RECONSTRUCTION_FAILED
+        result_object._error = "Reconstruction failed"
+        result_object._end_time = datetime.now(timezone.utc)
+        upsert._lattice_data(result_object)
+        await result_webhook.send_update(result_object)
+
+        return result_object
```

Comment:
@kessler-frost any tips for a test that would cover this `except` block? Tried patching the `_reconstruct` function to raise an error, but that puts me in the block below...

Reply:
Assuming this is necessary, based on `_postprocess_workflow`

Reply:
You can just deliberately fail the test task (say by doing a division by zero) and check if the result object is as expected. This will make it go to the `except` block.